styles.css

```diff
@@ -0,0 +1,236 @@
+* {
+  font-family: 'Open Sans', sans-serif;
+}
+
+header {
+  display: flex;
+  background-color: rgb(0, 51, 102);
+  width: 100%;
+  padding: 5% 3% 3% 8%;
+  border: none;
+  margin: 0;
+  justify-content: space-between;
+}
+
+h1 {
+  font-family: 'Ibarra Real Nova', serif;
+  font-style: italic;
+  font-size: 12pt;
+  color: rgb(0, 179, 60);
+}
+
+h2 {
+  font-family: 'Ibarra Real Nova', serif;
+}
+
+h3 {
+  font-family: 'Ibarra Real Nova', serif;
+}
+
+h4 {
+  margin-left: 1rem;
+  margin-right: 1rem;
+}
+
+.card {
+  margin-left: 1rem;
+  margin-right: 1rem;
+}
+
+nav {
+  width: 100%;
+  background-color: rgb(240, 240, 240);
+  padding-left: 3rem;
+  border: none;
+  margin-bottom: 1rem;
+  float: left;
+}
+
+div {
```

Comment:
Be careful with general selectors like this. You might not need position: relative on every div